frontend/src/lib/types.ts

```diff
@@ -0,0 +1,86 @@
+export interface ProjectDataType {
+    active_projects_count: number;
+    projects: project[]
+}
+
+export type project = {
+    idx_top_contributors: {
+        avatar_url: string;
+        contributions_count: number;
+        login: string;
+        name: string;
+    }[];
+    idx_contributors_count: number;
+    idx_forks_count: number;
+    idx_leaders: string[];
+    idx_level: string;
+    idx_name: string;
+    idx_stars_count: number;
+    idx_summary: string;
+    idx_topics: string[];
+    idx_type: string;
+    idx_updated_at: number; // UNIX timestamp
+    idx_url: string;
```

Comment:
I'd prefer to get rid of `idx_` prefix in interfaces if possible. It'd be more readable.
Do you think it's something that can be done?